runtime/src/execute_cost_table.rs

```diff
@@ -106,6 +106,17 @@ impl ExecuteCostTable {
             }
         }
 
+        // Logs events of unusual input `value` or `aggregated.ema`
+        const MAX_COST: u64 = 200_000;
+        if let Some(aggregated) = self.table.get(key) {
+            if value > MAX_COST || aggregated.ema > MAX_COST as f64 {
```

Comment:
this will miss the more interesting events, which are arithmetic under/overflow

Reply:
Thought capturing abnormal numbers are a good start, could log all, but would be too many lines.